### ISSUE TITLE
Blueprint mediapicker  not displaying images in pages - 404 media.json

### ISSUE THREAD

**Issue by sebastianbaumann** [getgrav/grav-plugin-admin#1457]
Hey guys,

just tried the new  `mediapicker` type in my page blueprint. 
If I open the modal the first time, it displays all images, but if I use the page navigation on the left, no images are shown at all, even if you reset the path via the `X` close button.

Snippet from my blueprint yaml file.
I tried changing the folder location to `@self`,  does not work either.

By the way... I couldn't find any information for what `folder` is used for.

```
  fields:
            picker:
              label: Bild auswählen
              type: mediapicker
              folder: '@page@:/media'
              preview_images: true
```

Following error is displayed in the chrome dev tools console:
 `Failed to load resource: the server responded with a status of 404 (Not Found) /admin/media.json/tmpl:media-list-content/index:0/page:%5Chome%5C_categories` 

![image](https://user-images.githubusercontent.com/9118735/40837125-9a6c1c8a-6599-11e8-9f08-9af612a0ebec.png)

We tested this on two complete different grav installations, same error each time.

**Comment by rhukster** [comment]
My test example looks like this:

```
           picker:
                  type: mediapicker
                  folder: 'self@'
                  label: Select a file
                  preview_images: true
                  accept:
                    - .jpg
```

**Comment by sebastianbaumann** [comment]
Ok, just tried to reproduce on my mac at home, works flawless. Could this be some local windows development issue, due the different file systems - regarding that 404 error for the `media.json` file?

**Comment by mahagr** [comment]
I agree that this may be because of differences between Windows and Mac/Linux..?

**Comment by mahagr** [comment]
I think this one is fixed in #1465
